index.html

```diff
@@ -381,10 +381,10 @@ <h3>Information for User Agents</h3>
 	      <li>Implement the host language method for keyboard navigation so that widgets that support <code>aria-activedescendant</code> may be included in the tab order.</li>
 	      <li>For platforms that expose <a>desktop focus</a> or <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> focus separately from <abbr title="Document Object Model">DOM</abbr> focus, do not expose the focused state in the accessibility <abbr title="application programming interface">API</abbr> for any element when it has <abbr title="Document Object Model">DOM</abbr> focus and also has <pref>aria-activedescendant</pref> which points to a valid <a href="#valuetype_idref">ID reference</a>.</li>
 	      <li>When the <pref>aria-activedescendant</pref> attribute changes on an element that currently has <abbr title="Document Object Model">DOM</abbr> focus, remove the focused state from the previously focused object and fire an accessibility <abbr title="application programming interface">API</abbr> <a>desktop focus event</a> on the new element referenced by <code>aria-activedescendant</code>. If <pref>aria-activedescendant</pref> is cleared or does not point to an element in the current document, fire a desktop focus event for the <a>object</a> that had the attribute change.</li>
-	      <li>For any element with an ID attribute, where the element is a descendant of an element with the <pref>aria-activedescendant</pref> attribute, apply the following accessibility <abbr title="Application Programming Interface">API</abbr> states to the target to ensure the object is accessible:
+	      <li>Apply the following accessibility <abbr title="Application Programming Interface">API</abbr> states to any element with an ID attribute that can be referenced by an  <pref>aria-activedescendant</pref> attribute. There are two ways an element can be referenced by the  <pref>aria-activedescendant</pref> attribute.  One way is when it is <a>owned</a> by an element with the <pref>aria-activedescendant</pref> and the other is when it is <a>owned</a> by an element that is controlled by an element with the <pref>aria-activedescendant</pref> attribute (e.g. see <ref>combobox</ref>):
```

Comment:
I'll defer to @aleventhal, @asurkov, and @jcsteh on this and the similar substantive changes below. The original editorial changes seem fine to me.

Reply:
@aleventhal wrote:
> > the other is when it is owned by an element that is controlled by an element with the aria-activedescendant attribute (e.g. see combobox)
> 
> * At least in Chrome, it's true that in this case the controlled by element needs to be one ancestor level up. Not sure if that's a bug or a useful optimization. It doesn't seem to make sense to overdo these checks, and look at every ancestor to see if there is an aria-controls pointing to it.

One ancestor up in DOM tree or AX tree? If it is an option in a controled listbox, you are looking for the parent listbox. Would it matter if there were a div around the option? If it is a button in a gridcell in a controled grid, you are looking for the parent grid. Would it matter if there were divs between the button and the gridcel or the gridcel and the grid? Since there are only 4 types of elements that can be controlled by a combobox or textbox, I wonder if Chrome shouldn't just look for the appropriate required context.

A controled grid is most tricky because you can put any kind of element inside a gridcell. And, the element inside the gridcel that is referenced by aria-activedescendant could then be anything and may be a few divs or spans deep inside the gridcell.

> * Also, Chrome requires that the element with aria-controls by a text box. This only matters for the focusable state, not for focus events. For focus events, Chrome is not strict and will follow the aria-activedescendant from any focused element to any other element.

should be textbox, searchbox, or combobox as specified in the aria-activedescendant definition.